CHANGELOG.md

```diff
@@ -65,7 +69,9 @@ and this project adheres to
   If you're upgrading to Teleplug 1.1.0, it is therefore recommended to also
   upgrade to OpenTelemetry API 1.1.0
 
-[Unreleased]: https://github.com/primait/teleplug/compare/1.1.3...HEAD
+
+[Unreleased]: https://github.com/primait/teleplug/compare/2.0.0...HEAD
+[2.0.0]: https://github.com/primait/teleplug/compare/1.1.3-rc.0...2.0.0
```

Comment:
is there a reason why we start from `1.1.3-rc.0` and not from `1.1.3`? is https://github.com/primait/teleplug/compare/1.1.3...1.1.3-rc.0 just noise?

Reply:
Mh that's weird, the update to the CHANGELOG happens automatically through `suite-py bump` and something weird happened here, nice find